indra/newview/llreflectionmap.cpp

```diff
@@ -74,18 +74,14 @@ void LLReflectionMap::autoAdjustOrigin()
     {
         const LLVector4a* bounds = mGroup->getBounds();
         auto* node = mGroup->getOctreeNode();
```

Comment:
This variable looks unused, mac build can complain about it.

Reply:
Sorry, 'node' check removal wasn't supposed to get into this commit, fixed.